doc/README-pt-br.md

```diff
@@ -0,0 +1,280 @@
+
+Não esqueça o `enctype="multipart/form-data"` em seu formulário.
+
+```html
+<form action="/profile" method="post" enctype="multipart/form-data">
+  <input type="file" name="avatar" />
+</form>
+```
+
+```javascript
+var express = require('express')
+var multer  = require('multer')
+var upload = multer({ dest: 'uploads/' })
+
+var app = express()
+
+app.post('/profile', upload.single('avatar'), function (req, res, next) {
+  // req.file is the `avatar` file
+  // req.body will hold the text fields, if there were any
+})
+
+app.post('/photos/upload', upload.array('photos', 12), function (req, res, next) {
+  // req.files is array of `photos` files
+  // req.body will contain the text fields, if there were any
+})
+
+var cpUpload = upload.fields([{ name: 'avatar', maxCount: 1 }, { name: 'gallery', maxCount: 8 }])
+app.post('/cool-profile', cpUpload, function (req, res, next) {
+  // req.files is an object (String -> Array) where fieldname is the key, and the value is array of files
+  //
+  // e.g.
+  //  req.files['avatar'][0] -> File
+  //  req.files['gallery'] -> Array
+  //
+  // req.body will contain the text fields, if there were any
+})
+```
+
+Caso você precise lidar com forumário text-only multipart, você deve usar o método `.none()`:
+
+```javascript
+var express = require('express')
+var app = express()
+var multer  = require('multer')
+var upload = multer()
+
+app.post('/profile', upload.none(), function (req, res, next) {
+  // req.body contains the text fields
+})
+```
+
+## API
+
+### Informação de arquivo
+
+Cada arquivo contém as seguintes informações:
+
+Key | Descrição | Nota
+--- | --- | ---
+`fieldname` | Nome do campo especificado no formulário |
+`originalname` | Nome do arquivo no computador do usuário |
+`encoding` | Tipo de codificação do arquivo |
+`mimetype` | Tipo Mime do arquivo |
+`size` | Tamanho do arquivo em bytes |
+`destination` | A pasta na qual o arquivo foi salvo | `DiskStorage`
+`filename` | O nome do arquivo dentro do `destination` | `DiskStorage`
+`path` | O caminho completo para o arquivo enviado | `DiskStorage`
+`buffer` | O `Buffer` do arquivo inteiro | `MemoryStorage`
+
+### `multer(opts)`
+
+Multer aceita um objeto de opções, a propriedade mais básico dos é o`dest`, que diz ao Multer onde fazer o upload dos arquivos. No caso de você omitir o objeto de opções, os arquivos serão mantidos na memória e nunca gravados no disco.
+
+Por padrão, Multer irá renomear os arquivos para evitar conflitos de nomes. A função de renomeação pode ser personalizada de acordo com suas necessidades.
+
+A seguir estão as opções que podem ser passadas para Multer.
+
+Key | Descrição
+--- | ---
+`dest` ou `storage` | Onde armazenar os arquivos
+`fileFilter` | Função para controlar quais arquivos são aceitos
+`limits` | Limites dos dados enviados
+`preservePath` | Mantenha o caminho completo dos arquivos em vez de apenas o nome base
+
+Em um web app basico, somente `dest` pode ser necessário, e configurado como mostrado em
```

Comment:
`básico` instead of `basico` 

`mostrado no exemplo a seguir` instead of `mostrado em o exemplo a seguir.`